src/world/common/SetNpcAnim.inc.c

```diff
@@ -0,0 +1,11 @@
+#include "common.h"
+
+static void SetNpcAnim(ScriptInstance* script) {
```

Comment:
`void`? If this is a script API function, the signature should be `ApiStatus SetNpcAnim(ScriptInstance*, s32)`. If it isn't, it should be called `set_script_owner_npc_anim` or similar.

Reply:
(`CamelCase` is reserved for script API funcs only!)

Reply:
Yeah, this should have a script function signature and return an ApiStatus.
![image](https://user-images.githubusercontent.com/9429556/97932559-3f419900-1d68-11eb-9fef-dbe95cff0ed2.png)

Reply:
Scratch that, this is **also not a script API function!** It's again called as part of 802403D4.